hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/validator/SparkPreCommitValidator.java

```diff
@@ -118,4 +120,11 @@ public HoodieEngineContext getEngineContext() {
   public HoodieWriteConfig getWriteConfig() {
     return this.writeConfig;
   }
+
+  protected Dataset<Row> executeSqlQuery(SQLContext sqlContext, String sqlQuery, String tableName) {
+    String queryWithTempTableName = sqlQuery.replaceAll(
+        HoodiePreCommitValidatorConfig.VALIDATOR_TABLE_VARIABLE, tableName);
+    LOG.debug("Running query: " + queryWithTempTableName);
+    return sqlContext.sql(queryWithTempTableName);
```

Comment:
Do we need another logging, the caller already logs one message.

Reply:
+1

Reply:
The caller does not log the query itself, so the query is printed here.  I can remove the caller logging.

Reply:
Addressed.